core/driver/tests/src/test/java/org/virtuslab/ideprobe/ProbeDriverJavaTest.java

```diff
@@ -1,40 +1,23 @@
 package org.virtuslab.ideprobe;
 
 import org.junit.Test;
-import org.virtuslab.ideprobe.ide.intellij.InstalledIntelliJ;
-import org.virtuslab.ideprobe.ide.intellij.IntelliJProvider;
 import org.virtuslab.ideprobe.protocol.Module;
 import org.virtuslab.ideprobe.protocol.ProjectRef;
-import scala.Function2;
 import scala.concurrent.duration.Duration;
-import scala.runtime.BoxedUnit;
-
-import java.nio.file.Path;
-import java.util.ArrayList;
 
 import static java.util.Arrays.asList;
 import static java.util.concurrent.TimeUnit.MINUTES;
 import static java.util.stream.Collectors.toList;
 import static org.junit.Assert.assertEquals;
 import static org.virtuslab.ideprobe.WaitLogic.emptyBackgroundTasks;
-import static org.virtuslab.ideprobe.WorkspaceTemplate.FromResource;
 import static org.virtuslab.ideprobe.wait.WaitLogicFactory.*;
-import static scala.collection.JavaConverters.collectionAsScalaIterable;
 import static scala.collection.JavaConverters.seqAsJavaList;
 
 public class ProbeDriverJavaTest {
     @Test
     public void openProject() {
-        var workspaceProvider = new FromResource("gradle-project");
```

Comment:
I am not sure about this change. You could add it as another test if you want. 
Both old and new version show how to setup IJFixture with a custom workspace from Java in 2 different ways (from code and from config).